mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/user/CassandraSubscriptionMapperTest.java

```diff
@@ -38,4 +39,9 @@ protected SubscriptionMapper createSubscriptionMapper() {
     public void tearDown() {
         CLUSTER.clearAllTables();
     }
+
+    @AfterClass
+    public static void stop() {
+        CLUSTER.close();
```

Comment:
Ah ah, that's funny because I told that about the cassandra server instance and here it's actually the cassandra client driver that happens to be named Cluster in datastax driver.
I still think it's wrong and I'll fix this field but still, it's not the same thing !